testrunner-maven-plugin/src/main/scala/edu/illinois/cs/testrunner/mavenplugin/TestRunner.scala

```diff
@@ -17,14 +18,14 @@ import scala.collection.JavaConverters._
 class TestRunner extends TestPlugin {
     def tests(source: String, project: MavenProject): Stream[String] = {
         if (source == null) {
-            TestLocator.tests(project)
+            TestLocator.tests(project, JUnit)
```

Comment:
Based on these changes, is it intentional that this TestRunner only applies to JUnit 4 tests? Should it be changed to handle JUnit 5 as well?

Reply:
It seems a legacy code. Based on the current usage of iDFlakies, this class (TestRunner) will not be used. The real class being used is DetectorPlugin.

I change this line to pass compile because I modify this signature of TestLocator.tests.

Not very sure what to do with this class.

Reply:
This seems to be a sample testrunner that simply runs the tests it can find within the project. Given the changes, right now it only runs JUnit 4 tests and does not run any JUnit 5 tests. To support JUnit 5, maybe add some code that tries both JUnit and JUnit5 as the parameter, so if it fails to get tests using one TestFramework it will try the other.

Reply:
Done. Please resolve this conversation if it looks good to you.